src/main/java/com/alkemy/ong/entity/ContactEntity.java

```diff
@@ -0,0 +1,47 @@
+package com.alkemy.ong.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@Entity
+@Table(name = "contact")
+@Getter
+@Setter
+@SQLDelete(sql = "UPDATE contact SET deletedAt = true WHERE id = ?")
+@Where(clause = "deletedAt = false ")
+public class ContactEntity {
+
+    @Id
+    @GeneratedValue(generator = "uuid")
+    @GenericGenerator(name= "uuid", strategy = "uuid2")
+    @Column(name = "id", nullable = false)
```

Comment:
In the previous entities we have name all the tables in lower case, just to be consistents